tools/harness-automation/autothreadharness/harness_case.py

```diff
@@ -342,7 +350,7 @@ def _setup_page(self):
         if not self.started:
             self.started = time.time()
 
-        if time.time() - self.started > 30:
+        if time.time() - self.started > 150:
```

Comment:
In case of timeout script refreshes the webpage and returns from this function. Right after, the `_setup_page` function is called again and since `self.started` value was not updated, we will end up in infinite loop of browser refreshing. I suggest to add follwoing line: `self.started = time.time()` inside the if statement.

Also i saw situation, where one of the sniffer didn't enumerate on windows correctly after boot up, i think in that case we will never finish the test case, because we will end up in inifinite loop in `test` function. I would really like to see an option to call `PDU.reboot` after such failures, or simply add some exit timeout.

Reply:
we can run the script on our testbeds , this commit is mainly for supporting mixed devices testbed which will be updated on https://openthread.io/certification/automation_setup soon. According to the issue, we will investigate it and continue optimizing the auto harness tool. Thanks a lot for your comments.